### PR TITLE
[JN-766] Remove all uses of var

### PR DESCRIPTION
**Pull request by connorlbark**
#### DESCRIPTION

More cleanup - just removes all usages of `var` in our codebase with the explicit type instead.

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

- All tests run / compiles ok
- No functional changes

### CODE REVIEW

File: core/src/main/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClient.java [hunk 1]
```diff
@@ -69,8 +71,8 @@ public PepperKit fetchKitStatus(UUID kitRequestId) throws PepperApiException, Pe
 
     @Override
     public Collection<PepperKit> fetchKitStatusByStudy(String studyShortcode) throws PepperApiException, PepperParseException {
-        var request = buildAuthedGetRequest("kitstatus/study/%s".formatted(makePepperStudyName(studyShortcode)));
-        var response = retrieveAndDeserializeResponse(request, PepperKitStatusResponse.class);
+        WebClient.RequestHeadersSpec<? extends WebClient.RequestHeadersSpec<?>> request = buildAuthedGetRequest("kitstatus/study/%s" .formatted(makePepperStudyName(studyShortcode)));
```
Comment (connorlbark) [inline core/src/main/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClient.java:74]: Not sure how I feel about this auto generated type, but I think it's still better than `var`.

File: core/src/main/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClient.java [hunk 2]
```diff
@@ -58,8 +60,8 @@ public PepperKit sendKitRequest(String studyShortcode, Enrollee enrollee, KitReq
 
     @Override
     public PepperKit fetchKitStatus(UUID kitRequestId) throws PepperApiException, PepperParseException {
-        var request = buildAuthedGetRequest("kitstatus/juniperKit/%s".formatted(kitRequestId));
-        var response = retrieveAndDeserializeResponse(request, PepperKitStatusResponse.class);
+        WebClient.RequestHeadersSpec<? extends WebClient.RequestHeadersSpec<?>> request = buildAuthedGetRequest("kitstatus/juniperKit/%s" .formatted(kitRequestId));
```
Comment (connorlbark) [inline core/src/main/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClient.java:63]: Not sure how I feel about this, but I think I still prefer over var.

**Review by devonbush — APPROVED** [review]
nice!  let's get this merged FAST before merge conflicts start piling up

**Review by MatthewBemis — APPROVED** [review]
Thanks for doing this! I mostly skim-reviewed for the sake of unblocking the merge, but looks legit!

**Comment by devonbush** [comment]
merging fast to avoid merge conflicts

**Pull request merged**